barretenberg/cpp/src/barretenberg/aztec_ivc/aztec_ivc.cpp

```diff
@@ -38,7 +11,7 @@ void AztecIVC::accumulate(ClientCircuit& circuit, const std::shared_ptr<Verifica
  */
 void AztecIVC::complete_kernel_circuit_logic(ClientCircuit& circuit)
 {
-    BB_OP_COUNT_TIME_NAME("construct_circuits");
```

Comment:
I guess this no longer takes significant time?

Reply:
Well it's still measured but its measured all at once as part of circuit construction in the benchmark. Maybe its better to keep it here and add BB_OP_COUNT_TIME_NAME("construct_circuits"); to the mock circuit methods individually

Reply:
Oh but I guess I didn't do this because it threw off the bench because the circuits are constructed twice sometimes for the precomputed vk stuff